core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/runtimeModels.scala

```diff
@@ -22,7 +22,7 @@ final case class Runtime(id: Long,
                          internalId: RuntimeInternalId,
                          runtimeName: RuntimeName,
                          googleProject: GoogleProject,
-                         serviceAccountInfo: ServiceAccountInfo,
+                         serviceAccount: WorkbenchEmail,
```

Comment:
I'm seeing `serviceAccount`, `googleServiceAccount`, and `clusterServiceAccount` - are these all the same field? I think it would be clearer if the naming was consistent

Reply:
yea...they're the same. `googleServiceAccount` is swagger doc, and I made json encoder that way to match swagger. `clusterServiceAccount` is how it's called in database, I guess I can rename the column in database as well